src/libcore/iter/adapters/mod.rs

```diff
@@ -504,6 +504,9 @@ where
 #[stable(feature = "fused", since = "1.26.0")]
 impl<I> FusedIterator for Cycle<I> where I: Clone + Iterator {}
 
+#[unstable(feature = "trusted_len", issue = "37572")]
```

Comment:
There is no such thing as an unstable trait impl so this would need to be changed to #\[stable\].

Reply:
`TrustedLen` isn't a stable trait.